### DISCUSSION TITLE
Iron ore fines 62% Fe CFR Futures

### DISCUSSION THREAD

**Discussion by maykon17** [alvarobartt/investpy#406] [Q&A]
Hi! I'm currently extracting historical market data and investpy is very useful to me, but the commodity "Iron ore fines 62% Fe CFR Futures" doesn't seem to be available in investpy, I didn't find the historical data and the smybol in commodities list. Could anyone help me? Thanks. #323

**Comment by knickr** [comment]
Its in investpy and investing.com

search_result = investpy.search_quotes(text='Iron ore', products=['commodities'], countries=['united states'], n_results=1)

{"id_": 961729, "name": "Iron ore fines 62% Fe CFR Futures", "symbol": "TIOc1", "country": null, "tag": "/commodities/iron-ore-62-cfr-futures", "pair_type": "commodities", "exchange": "CME"}

you can then pull historical data

historical_data = search_result.retrieve_historical_data(from_date='01/01/2019', to_date='18/08/2021')
